plugin/plugin.cpp

```diff
@@ -12,15 +12,15 @@
 class Q_DECL_EXPORT QrFilterPlugin : public QQmlExtensionPlugin
 {
     Q_OBJECT
-    Q_PLUGIN_METADATA(IID "org.sailfishos.qrfilter")
+    Q_PLUGIN_METADATA(IID "amber.qrfilter")
 
 public:
     QrFilterPlugin() = default;
     virtual ~QrFilterPlugin() = default;
 
     void registerTypes(const char *uri) override
     {
-        Q_ASSERT(QLatin1String(uri) == QStringLiteral("org.sailfishos.qrfilter"));
+        Q_ASSERT(QLatin1String(uri) == QStringLiteral("amber.qrfilter"));
```

Comment:
Wait a second, shouldn't this be Amber.QrFilter? Or should have been already uppercase sailfish for what i can tell.

Reply:
Good catch

Reply:
So, `Amber.QrFilter` should be instead?

Reply:
Fixed, thx

Reply:
Pekka, thanks